pkg/drift_client/drift_client.py

```diff
@@ -96,18 +96,19 @@ def get_list(self, topics: List[str], timeframe: List[str]) -> Dict[str, List[st
             >>> # => {"topic-1": ['topic-1/1644750600291.dp',
             >>> #                  'topic-1/1644750601291.dp', ...] ... }
         """
-        to_request = self.__influx_client.query_data(topics, timeframe[0], timeframe[1])
-
         data = {}
+        for topic in topics:
+            to_request = self.__influx_client.query_data(
```

Comment:
what does "to_request" mean?

Reply:
a good question. I think it should have name `values`

Reply:
@aschenbecherwespe renamed